components/engine_rocks/src/config.rs

```diff
@@ -5,6 +5,10 @@ use rocksdb::{DBCompressionType, DBInfoLogLevel, DBTitanDBBlobRunMode};
 use serde::{Deserialize, Serialize};
 use std::str::FromStr;
 
+pub use rocksdb::{CompactionPriority, DBCompactionStyle, DBRateLimiterMode, DBRecoveryMode};
```

Comment:
And remove the pub use.

Reply:
https://github.com/tikv/tikv/blob/8017333565f3525fcb49563590cd13375e596e58/components/engine_traits/src/config.rs#L3

Sounds we should also change here